### PR TITLE
fixes #582 (Android not setting this onLayout first?)

### LINKED ISSUE #582: Cannot read property 'x' of undefined
### Version
Which versions are you using:

- react-native-swiper ^1.5.12
- react-native 0.48.3

### Expected behaviour
Android device debugging will not be throw error

### Actual behaviour
Android device debugging is throw error Cannot read property 'x' of undefined, 
When I Debug JS Remotely to chrome was Report errors,
It's normal after stop remote js debugging

### PR DESCRIPTION
**Pull request by fungilation**
Fixes #582

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -619,6 +621,16 @@ export default class extends Component {
     this.scrollView = view;
   }
 
+  onPageScrollStateChanged = state => {
+    switch (state) {
+      case 'dragging':
+        return this.onScrollBegin();
+        case 'idle':
```
Comment (arribbar) [inline src/index.js]: You should unindent this line no ?

**Comment by arribbar** [comment]
I have just merged #684. Please, resolve conflicts and I'll do a new release within this week

**Comment by fungilation** [comment]
Styling fixed

**Comment by arribbar** [comment]
It seems you still have conflicts since I have merged #684. Please resolve them :)

**Comment by fungilation** [comment]
Call me a github noob, but how do I resolve conflicts directly? The button "resolve conflicts" is readonly for me, with this block for the conflict:
```
<<<<<<< master
=======

>>>>>>> master
```

which is a difference of a line break. You can merge with or without?

**Comment by arribbar** [comment]
@fungilation, no worries ;)
I am not very familiar working with forks but you cannot do it here as you have only read rights. You should be able to do it from your fork. Else, you can still do it in local...

**Comment by fungilation** [comment]
Here you go, no more conflicts

**Pull request merged**